Observers/NotifyAdmin.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Inchoo\FAQNotification\Observers;
+
+use Magento\Catalog\Model\ProductRepository;
+use Magento\Customer\Model\Session;
+use Magento\Framework\Escaper;
+use Magento\Framework\Event\Observer;
+use Magento\Framework\Mail\Template\TransportBuilder;
+use Magento\Store\Model\StoreManagerInterface;
+
+class NotifyAdmin implements \Magento\Framework\Event\ObserverInterface
+{
+    const ADMIN_NOTIFICATION_EMAIL = 'admin@example.com';
+
+    public function __construct(
+        Session $customerSession,
+        ProductRepository $productRepository,
+        TransportBuilder $transportBuilder,
+        Escaper $escaper,
+        StoreManagerInterface $storeManager
+    ) {
+        $this->customerSession = $customerSession;
+        $this->productRepository = $productRepository;
+        $this->transportBuilder = $transportBuilder;
+        $this->escaper = $escaper;
+        $this->storeManager = $storeManager;
+    }
+
+    public function execute(Observer $observer)
+    {
+        $eventData = $observer->getData('question');
```

Comment:
$eventData ime varijable mi baš i ne govori što je to

Reply:
ime varijable promijenjeno da bolje odgovara opisom